internal/testplanet/planet.go

```diff
@@ -517,12 +518,13 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				PasswordCost:    console.TestPasswordCost,
 				AuthTokenSecret: "my-suppa-secret-key",
 			},
+			Vouchers: satVouchers.Config{
+				Expiration: 30,
```

Comment:
is this a time.Duration? if it is, then this means 30 nanoseconds. if it's not a time.Duration, can we make it one, and then change this to 30*24*time.Day?

Reply:
ok